SevenSlots/SevenSlots/SevenSlots/View/HeaderContentView.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Android.Provider;
+﻿//using Android.Provider;
```

Comment:
what's with all the commenting??

Reply:
Meant to ask you about this import, but then I saw the program working without them as well and just forgot about them. I'll delete them.